src/Tizen.NUI.Scene3D/src/public/ModelMotion/MotionValue.cs

```diff
@@ -152,7 +152,7 @@ public KeyFrames KeyFramesValue
         /// <summary>
         /// Get the type of value what we set.
```

Comment:
"what we set" -> "which we set", "controlled by the motion", "animated with the MotionValue", same in line 165

Reply:
Change as "which we set"